### PR TITLE
[Examples] Fix Out-of-memory in PAL

### PR DESCRIPTION
**Pull request by sprzysta**
Signed-off-by: Przystawski <sebastian.przystawski@intel.com>

Without specifying Graphene metadata size this pytorch sample application
was not able to run because of "Out-of-memory in PAL" error.
Specifying "loader.pal_internal_mem_size" in manifest fix that issue.

 Changes to be committed:
	modified:   Examples/pytorch/pytorch.manifest.template

## Description of the changes
Added loader.pal_internal_mem_size = "128M" to pytorch manifest.template

## How to test this PR?
Buld and run pytorch example according to the instructions
in https://github.com/sprzysta/graphene/blob/master/Examples/pytorch/README.md.
Example application should run without error

---
This change is [Reviewable](https://reviewable.io/reviews/oscarlab/graphene/2512)

### CODE REVIEW

**Review by dimakuv — APPROVED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene/2512)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel)

*[a discussion](https://reviewable.io/reviews/oscarlab/graphene/2512#-MdvasE392HmHsptMeKn:-MdvasE392HmHsptMeKo:bkyabci) (no related file):*
Thanks for the fix! Looks like a no-brainer to merge.

---

**Comment by dimakuv** [comment]
Jenkins, test this please

**Review by mkow — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene/2512)* status: all files reviewed, 2 unresolved discussions, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel) (waiting on @sprzysta)

*[a discussion](https://reviewable.io/reviews/oscarlab/graphene/2512#-MdyPeEMDJ6Jw_Dt5IVs:-MdyPeEN28QejvlyP5qN:bqjzu1g) (no related file):*
> Signed-off-by: Przystawski <sebastian.przystawski@intel.com>

Hmm, how have you inserted this? `git --signoff` appends the sign-offs at the end of the commit message, and that's what you should be doing :)

---
*[a discussion](https://reviewable.io/reviews/oscarlab/graphene/2512#-MdyQ3Oc08WuSmEDdIZi:-MdyQ3Oc08WuSmEDdIZj:b-9q97by) (no related file):*
>  Changes to be committed:
> 	modified:   Examples/pytorch/pytorch.manifest.template

Why have you added this to the commit message?

---

**Review by dimakuv — DISMISSED** [review]
The original committer timed out, so I rebased this commit according to Michal's comments.

> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene/2512#-:-MeiONWu1ReMSDQQMf6m:bg67fun)* status: all files reviewed, 2 unresolved discussions, not enough approvals from maintainers (1 more required), not enough approvals from different teams (1 more required, approved so far: Intel) (waiting on @mkow)

*[a discussion](https://reviewable.io/reviews/oscarlab/graphene/2512#-MdyPeEMDJ6Jw_Dt5IVs:-MeiOM0G19KdWHxgVFr9:b-896fix) (no related file):*
<details><summary><i>Previously, mkow (Michał Kowalczyk) wrote…</i></summary><blockquote>

> Signed-off-by: Przystawski <sebastian.przystawski@intel.com>

Hmm, how have you inserted this? `git --signoff` appends the sign-offs at the end of the commit message, and that's what you should be doing :)
</blockquote></details>

Done.

---
*[a discussion](https://reviewable.io/reviews/oscarlab/graphene/2512#-MdyQ3Oc08WuSmEDdIZi:-MeiOMcKCpCfaezu8_A6:b-896fix) (no related file):*
<details><summary><i>Previously, mkow (Michał Kowalczyk) wrote…</i></summary><blockquote>

>  Changes to be committed:
> 	modified:   Examples/pytorch/pytorch.manifest.template

Why have you added this to the commit message?
</blockquote></details>

Done.

---

**Review by dimakuv — APPROVED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene/2512)* status: all files reviewed, 2 unresolved discussions, not enough approvals from maintainers (1 more required), not enough approvals from different teams (1 more required, approved so far: Intel) (waiting on @mkow)

**Comment by dimakuv** [comment]
Jenkins, test this please

**Pull request merged**
